lib/js/src/compose/types/page-block/metric.ts

```diff
@@ -1,16 +1,22 @@
 import { PageBlock, PageBlockInput, Registry } from './base'
 import { dimensionFunctions } from '../chart/util'
 import { CortezaID, Apply } from '../../../cast'
-
+import { Options as PageBlockRecordListOptions } from './record-list'
 const kind = 'Metric'
 
 type Reporter = (p: ReporterParams) => Promise<any>
 
 interface DrillDown {
   enabled: boolean;
   blockID?: string;
+  recordListOptions: Partial<PageBlockRecordListOptions>;
 }
 
+const drillDownDefaults: Readonly<DrillDown> = Object.freeze({
+  enabled: false,
```

Comment:
also add blockID: undefined in the defaults. Lets keep defaults and interface 1:1